package.json

```diff
@@ -2,11 +2,21 @@
   "name": "urql",
   "version": "1.0.5",
   "description": "A highly customizable and versatile GraphQL client for React",
+  "private": false,
+  "repository": "https://www.github.com/FormidableLabs/urql",
+  "bugs": "https://github.com/FormidableLabs/urql/issues",
   "main": "dist/urql.js",
   "module": "dist/urql.es.js",
   "types": "dist/types/index.d.ts",
   "source": "src/index.ts",
   "sideEffects": false,
+  "keywords": [
```

Comment:
Let's add `graphql client` as one word, `state management` as two, and add `exchanges` :+1:

Reply:
I think I got it right, not entirely sure got confused with one word and two. So I made graphql -> graphql client state-management -> state management and added exchanges